### PR TITLE
Setup and Mobile Version Skeleton Review

### PR DESCRIPTION
**Pull request by iammouaz**
Hello,

Kindly review my code for the first Portfolio that's list of what I did :
1-Add linters
2-Add Header
3-Add headline section
4-Add new section & style changes
5-Add icons section
6-Edit the README.md file

thanks a lot

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            
+</header>
+<section class="intro">
+<h1 class="big-name"> Mouaz El Molkey</h1>
+<img src="img/circle-1.png" class="circle-1" alt="">   
+<h2 class="small-name">Software Developer</h2>
+</section>
+
+<section class="main-section">
+    <h3 class="can-do">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</h3>
```
Comment (emmanuellekamwa) [inline index.html:32]: - [ ] This text should be wrapped inside a `p` tag and not an `h3` tag. Please fix that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+</li>
+</ul>
+</section>
+
+<section class="social">
+<img class="social-items" src="/img/github.png" alt="">
+<img class="social-items" src="/img/linkedin.png" alt="">
+<img class="social-items" src="/img/vic.png" alt="">
+<img class="social-items" src="/img/twitter.png" alt="">
+<img class="social-items" src="/img/micro.png" alt="">
```
Comment (emmanuellekamwa) [inline index.html:77]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Please fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @iammouaz 👋🏽
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Please your pull request is not descriptive enough. Try and add the changes that you have made to your project

_Check the comments under the review._👇🏽👀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,76 +1,21 @@
...
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
```
[the remark this answers is not in the document]
Comment (iammouaz) [reply README.md:41]: hello but I think I've already modify it in this brunch are you sure ??

File: README.md [hunk 4]
```diff
@@ -1,76 +1,21 @@
...
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Mouaz Molkey**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
```
[the remark this answers is not in the document]
Comment (iammouaz) [reply README.md:57]: hello but I think I've already modify it in this brunch are you sure ??

File: README.md [hunk 5]
```diff
@@ -1,76 +1,21 @@
...
-### Run tests
-
-### Deployment
-
-
+- HTML
+- CSS(Flexbox)
 
 ## Authors
```
Comment (carlos-ssh) [inline README.md:11]: Kindly provide information related to your project. keep the professionalism simple things make the difference.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,253 @@
+.body {
+  position: relative;
+  background: #191765;
+  display: grid;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
```
Comment (carlos-ssh) [inline css/style.css:7]: - Kindly add to buttons is `` <a> `` link a or button tag, and follow the button styling: Active, Hover and Pressed.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @iammouaz , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @iammouaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**